packages/bot-arbitrage/test/integration/arbBot.integration.test.ts

```diff
@@ -404,8 +627,151 @@ describe("ArbBot integration tests", () => {
         holdingDaiBeforeBalance < holdingWethfterBalance,
         "Should have gained holding token"
       );
+      /* check if the offer has been partially filled */
+      assert.equal(makerBaseAfterBalance.toNumber() > 99, true);
+      assert.equal(makerQuoteAfterBalance.toNumber() < 1, true);
+    });
+
+    it(`should find arb and do arb, bid`, async function () {
```

Comment:
Same name as the first test. Does it test the same thing?

Reply:
No it does a partial fill I addressed the naming :)